sdk/core/core-http/package.json

```
@@ -28,15 +28,24 @@
   ],
   "main": "./dist/coreHttp.node.js",
   "module": "./es/src/coreHttp.js",
-  "types": "./es/src/coreHttp.d.ts",
+  "types": "./types/src/coreHttp.d.ts",
+  "typesVersions": {
+    "<3.6": {
+      "*": [
+        "./types3.1/src/coreHttp.d.ts"
```

Should this be 
```suggestion
        "./types3.1/*"
```
Or is that not necessary due to using a rollup d.ts file?

---

Yes it should be *.  Roll up didn't work yet for me.